internal/cluster/cluster.go

```diff
@@ -265,7 +265,16 @@ func (m *Manager) ClusterMonitoring() {
 	defer func() {
 		m.prevMonitorInvokedAt = curMonitorInvokedAt
 	}()
-	if err = m.db.Find(&clusters).Error; err != nil {
+
+	//no need to refresh cluster status if the cluster is in the following statuses
+	noNeedToMonitorInStates := []string{
+		models.ClusterStatusPreparingForInstallation,
+		models.ClusterStatusFinalizing,
```

Comment:
Don't we have timeout for both `ClusterStatusPreparingForInstallation` `ClusterStatusFinalizing` ?

Reply:
thanks, fixed it